app/views/teams/show.html.erb

```diff
@@ -18,41 +17,80 @@
   <dd><%= @team.description %></dd>
 </dl>
 
-<table class="table table-striped">
-  <tbody>
+<%= form_tag perform_action_on_multiple_members_team_path do %>
+  <table class="table table-striped">
+    <tbody>
     <% @team.members.each do |member| %>
-      <tr>
-        <td>
-          <%= member.id%> <%= member.first_name%> <%= member.last_name%>
-            
-          <% if can? :delete_ownership, @team %>
-            <%= link_to t('.delete_ownership', :default => t("helpers.links.delete_ownership")),
-            delete_ownership_team_path(@team, :team_member => member.id), :method => 'post', :class => 'btn btn-default btn-xs' if @team.owners.include? member %>
-          <% end %>
-
-          <% if can? :assign_ownership, @team %>
-            <%= link_to t('.assign_ownership', :default => t("helpers.links.assign_ownership")),
-            assign_ownership_team_path(@team, :team_member => member.id), :method => 'post', :class => 'btn btn-default btn-xs' if !@team.owners.include? member  %>
-          <% end %>
-
-          <%= link_to t('.delete_membership', :default => t("helpers.links.delete_membership")),
-          delete_membership_team_path(@team, :team_member => member.id), :method => 'post', :class => 'btn btn-xs btn-danger' %>
-        </td>
-      </tr>
+        <tr>
+          <td><%= check_box_tag "members[]", member.id %></td>
+          <td><%= member.id %></td>
+          <td><%= member.first_name%></td>
+          <td><%= member.last_name%></td>
+          <td>
+            <%= link_to t('.email', :default => t("helpers.links.email")),
+                        team_path(@team),
```

Comment:
This url has to be adjusted, doesn’t it? I thought it should point to a mailto link?

Reply:
This is not part of this issue